manifests/charts/istio-cni/values.yaml

```
@@ -105,7 +105,7 @@ global:
 
   # change cni scope level to control logging out of istio-cni-node DaemonSet
   logging:
-    level: default:info,cni:info
+    level: default:info,cni:info,ambient:info,ebpf:info
```

```suggestion
    level: default:info,cni:info,ambient:info,ebpf:info
```

What about simply making `ebpf` follow the log level of `cni`, rather than adding another category?

That's simpler and will generally do what people expect, and `ebpf` isn't really a component in the same sense that `cni` is.

(`ambient` isn't either TBQH)

---

Yes, we could combine both ambient/ebpf into cni scope. Here is just adding a hint at deploy level and probably out of the scope of this PR, and we do have registered `cni` `ambient` `ebpf` scope in the code, and also it will been seen it in ctrlz scope.

---

`out of the scope of this PR` that's reasonable. It's just something that caught my eye.

---

- Can the level be updated at runtime ( including epbf ) ? 
- isn't info the default value for log level already ? If yes - can we remove this from values.yaml ? 

I'm not sure the best debug pattern is to change values and reinstall - enabling debug on all nodes. 
CNI is already a bit too verbose  (all the 'not enabling in NS x because' from each node), now that it's a proper server we could have a config_dump that shows all this on demand and maybe the log-control endpoint.

---

> * Can the level be updated at runtime ( including epbf ) ?

Yes, golang based log level could be updated at runtime (via ctrlz AFAIK)

> * isn't info the default value for log level already ? If yes - can we remove this from values.yaml ?

Yes, default level for any scope is `info` level ([func RegisterScope](https://github.com/istio/pkg/blob/53ca9b2f1a0a7192b4f1b35984f5b08cf38037f4/log/scope.go#L108)).  

Sure, I'll remove what I added for scope level in values.yaml in this PR.